### PR TITLE
Migrate transfers

### PR DESCRIPTION
**Pull request by SweetmanTech**

### CODE REVIEW

File: src/Cre8ors.sol [hunk 1]
```diff
@@ -479,7 +449,7 @@ contract Cre8ors is
                 revert ICre8ing.Cre8ing_Cre8ing();
             }
         }
-        ERC721AC._beforeTokenTransfers(from, to, startTokenId, quantity);
+        ERC721ACH._beforeTokenTransfers(from, to, startTokenId, quantity);
```
Comment (SweetmanTech) [inline src/Cre8ors.sol]: I think you can make this `super` 

```suggestion
        super._beforeTokenTransfers(from, to, startTokenId, quantity);
```

File: src/Transfers.sol [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        if (!hasRole(DEFAULT_ADMIN_ROLE, msg.sender)) {
+            revert Access_OnlyAdmin();
+        }
+
+        _;
+    }
+
+    function ownerOf(uint256 tokenId) public view returns (address) {
+        return baseContract.ownerOf(tokenId);
+    }
```
Comment (SweetmanTech) [inline src/Transfers.sol]: why do we need an `ownerOf` function on `Transfers.sol`?

Comment (SweetmanTech) [reply src/Transfers.sol]: Can we remove this?

File: src/interfaces/ICre8ors.sol [hunk 3]
```diff
@@ -19,4 +19,6 @@ interface ICre8ors is IERC721Drop, IERC721A {
     function cre8ing() external view returns (ICre8ing);
     /// @notice Getter for last minted token ID (gets next token id and subtracts 1)
     function _lastMintedTokenId() external view returns (uint256);
+
+    function ownerOf(uint256 tokenId) external view  returns (address);
```
Comment (SweetmanTech) [inline src/interfaces/ICre8ors.sol]: same question, do we need this for the work you're doing on hooks?

Comment (SweetmanTech) [reply src/interfaces/ICre8ors.sol]: or, can we remove this change?

File: src/Transfers.sol [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        uint256,
+        uint256
+    ) external view returns (bool) {
+        return afterTokenTransfersHookEnabled;
+    }
+
+    /// @notice Custom implementation for AfterTokenTransfers Hook.
+    function afterTokenTransfersOverrideHook(
+        address from,
+        address,
```
Comment (crazzywizard) [inline src/Transfers.sol:58]: curious why there is an unused function param here

File: test/minter/CollectionHolderMint.t.sol [hunk 5]
```diff
@@ -287,7 +287,7 @@ contract CollectionHolderMintTest is DSTest, StdUtils {
 
     function generateTokens(
         uint256 quantity
-    ) internal returns (uint256[] memory) {
+    ) internal pure returns (uint256[] memory) {
```
Comment (SweetmanTech) [inline test/minter/CollectionHolderMint.t.sol:290]: this is the only change. ✏️ 

everything else is formatting in this file.

**Review by SweetmanTech — COMMENTED** [review]
ship it 🚢

**Pull request merged**